code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -199,6 +205,7 @@
 	bolt_type = BOLT_TYPE_NO_BOLT
 	can_be_sawn_off  = TRUE
 	pb_knockback = 3 // it's a super shotgun!
+	manufacturer = MANUFACTURER_NONE
```

Comment:
Shouldn't this be Hunter's Pride? It's right up their alley.

Reply:
I always saw the double barreled shotgun as an indie weapon, like anyone could make one pretty easily, but yeah, I can make it HP